.github/workflows/ci.yaml

```diff
@@ -427,11 +497,6 @@ jobs:
       - name: diff-dumps
         run: ./scripts/ci/jobs/diff-dumps.sh
 
-      - uses: actions/upload-artifact@v4
-        with:
-          name: diff-dumps-inspect
-          path: /tmp/diff-dumps-inspect
```

Comment:
how come this was removed?

Reply:
good catch!  not sure what happened here...